src/hooks/UrqlUseMutation.re

```diff
@@ -22,12 +22,17 @@ external useMutationJs:
   (~query: string) => (useMutationResponseJs('a), executeMutation) =
   "useMutation";
 
-let useMutationResponseToRecord = (result: useMutationResponseJs('a)) => {
-  let data = result->dataGet;
+let useMutationResponseToRecord =
+    (parse: Js.Json.t => 'response, result: useMutationResponseJs('a)) => {
```

Comment:
Similar to how we did things on `useQuery`, I believe `useMutationResponseJs('a)` can lose the type parameter and we can just type `data` as `Js.Json.t`.